engine/sim/sc_profileset.hpp

```diff
@@ -418,7 +524,7 @@ void create_options( sim_t* sim );
 
 statistical_data_t collect( const extended_sample_data_t& c );
 statistical_data_t metric_data( const player_t* player, scale_metric_e metric );
-void save_output_data( std::unique_ptr<profile_set_t>& profileset, const player_t* parent_player, const player_t* player, std::string option );
+void save_output_data( profile_set_t* profileset, const player_t* parent_player, const player_t* player, std::string option );
```

Comment:
I would change the signature to 
void save_output_data( profile_set_t& profileset, const player_t* parent_player, const player_t* player, std::string option );

Reply:
Edit: Should have re-read what navv wrote, the exact same thing ;)